packages/http-smoke-testing/composer.json

```diff
@@ -21,15 +21,16 @@
         }
     },
     "require": {
-        "php": "^5.6|^7.0",
-        "phpunit/phpunit": "^3.7.15|^4.0|^5.0|^6.0|^7.0",
-        "symfony/framework-bundle": "^2.5|^3.0|^4.0",
-        "symfony/http-foundation": "^2.5|^3.0|^4.0",
-        "symfony/dependency-injection": "^2.5|^3.0|^4.0",
-        "symfony/routing": "^2.5|^3.0|^4.0"
+        "php": "^7.2",
+        "phpunit/phpunit": "^7.0",
+        "symfony/framework-bundle": "^3.4|^4.0",
+        "symfony/http-foundation": "^3.4|^4.0",
+        "symfony/dependency-injection": "^3.4|^4.0",
+        "symfony/routing": "^3.4|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.4.3|^6.0|^7.0",
+        "phpunit/phpunit": "^7.0",
```

Comment:
why we have phpunit in require and also in require-dev ?

Reply:
Removed from phpunit